tox.ini

```diff
@@ -13,21 +18,19 @@ deps =
     -e.
     -rrequirements/test.txt
     django18: Django>=1.8,<1.9
-    django110: Django>=1.10,<1.11
     django111: Django>=1.11,<1.12
 passenv =
     DISPLAY
 commands =
     make var/workbench.db
-    make test
+    py.test {posargs}
```

Comment:
"pytest" has been recommended over "py.test" since the 3.0.0 release.

Reply:
I'll update it here, but this actually came in from the cookiecutter template so we should fix it there too.